apps/designer/app/designer/features/style-panel/shared/use-style-data.ts

```diff
@@ -220,6 +251,8 @@ export const useStyleData = ({
 
   return {
     currentStyle,
+    setStyle: cssRule?.style ?? ({} as Style),
```

Comment:
setStyle looks like function name but its not

Reply:
Stupid word

Reply:
should we rename all "set" usages to "local"?

Reply:
localStyle, ownStyle, currentStyle, selectedStyle (for breakpoint and instance)

Reply:
@TrySound are those suggestions for the same thing?

Reply:
Yes